daffodil-cli/src/it/scala/org/apache/daffodil/parsing/TestCLIParsing.scala

```diff
@@ -1278,18 +1278,18 @@ class TestCLIparsing {
     }
   }
 
-  @Test def test_XXX_CLI_Parsing_SimpleParse_sax(): Unit = {
+  @Test def test_XXX_CLI_Parsing_SimpleParse_exi(): Unit = {
 
     val schemaFile = Util.daffodilPath("daffodil-test/src/test/resources/org/apache/daffodil/section00/general/generalSchema.dfdl.xsd")
     val (testSchemaFile) = if (Util.isWindows) (Util.cmdConvert(schemaFile)) else (schemaFile)
 
     val shell = Util.start("")
 
     try {
-      val cmd = String.format(Util.echoN("Hello") + "| %s parse -I sax -s %s -r e1", Util.binPath, testSchemaFile)
+      val cmd = String.format(Util.echoN("Hello") + "| %s parse -I exi -s %s -r e1 | md5sum", Util.binPath, testSchemaFile)
```

Comment:
I wonder if these will fail CI once we get passed the rat check?

Reply:
Yes, this would fail.  I have simply disabled this test on windows as there isn't a straightforward way to get a hash of the output of a command and I'm hesitant to deal with temporary files and cleaning them up without some sort of more secure API beyond just redirecting output to a file.